gardenscript.js

```diff
@@ -0,0 +1,15 @@
+$(document).ready(function(){
```

Comment:
Please see: https://codepen.io/abbeyjfitzgerald/pen/LexmVG

jQuery is to be avoided at all costs